db/migrate/20151106172531_remove_uniqueness_display_name_index_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniquenessDisplayNameIndexUsers < ActiveRecord::Migration
+  def change
+    remove_index :users, column: :display_name, unique: true
```

Comment:
I'm not sure that it's a real problem as we report the @user  handle and the display name together in talk.

![image](https://cloud.githubusercontent.com/assets/295329/11031625/b86d0c10-86cd-11e5-995e-ca6307c74cea.png)